src/dataflow/components/nodes/controls/value-control.tsx

```diff
@@ -15,7 +16,7 @@ export class ValueControl extends Rete.Control {
 
     this.component = (compProps: { value: number; sentence: string }) => (
       <div className="value-container">
-        {compProps.sentence ? compProps.sentence : compProps.value}
+        {compProps.sentence ? compProps.sentence : roundNodeValue(compProps.value)}
```

Comment:
Completely unrelated to this PR, but I realize I forgot to mention it on the PR where it would have been relevant -- "sentence" is not the right word for this. I think we should consider using something else ("expression"?) down the road.

Reply:
Yeah, I was thinking of the idea of a "number sentence", but "expression" is probably clearer.